scout/build/case.py

```diff
@@ -289,6 +289,7 @@ def build_case(case_data, adapter):
     case_obj["vcf_files"] = case_data.get("vcf_files", {})
     case_obj["omics_files"] = case_data.get("omics_files", {})
     case_obj["delivery_report"] = case_data.get("delivery_report")
+    case_obj["rna_delivery_report"] = case_data.get("rna_delivery_report")
```

Comment:
I have a feeling a lot of the build functionality could be replaced with serialisation aliases and a few functions for the pydantic case model..

Reply:
Mm yes. But another PR